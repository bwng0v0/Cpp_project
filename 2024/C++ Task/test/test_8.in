7
1
5
0
5
14
5
14
5
5
0
5
10
0
0
