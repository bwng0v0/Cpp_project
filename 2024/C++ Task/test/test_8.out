
******************************* Main Menu *********************************
* 0.Exit 1.PersonManager(ch3_2, 4, 6, 7_2, 8, 9, 10)                      *
* 2.Class:Object(ch3_1) 3.CopyConstructor(ch5_1) 4.AllocatedMember(ch5_2) *
* 5.OperatorOverload(ch7_1) 6.Inheritance(ch8) 7.PMbyVector               *
***************************************************************************
Menu item number? 7

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 1
display(): count 16
[0] p0 10 70 false :Gwangju Nam-gu Bongseon-dong 21:
[1] p1 11 61.1 true :Jong-ro 1-gil, Jongno-gu, Seoul:
[2] p2 12 52.2 false :1001, Jungang-daero, Yeonje-gu, Busan:
[3] p3 13 83.3 true :100 Dunsan-ro Seo-gu Daejeon:
[4] s1 21 65.4 true :Jongno-gu Seoul:
[5] s2 22 54.3 false :Yeonje-gu Busan:
[6] s3 23 55.5 true :Dong-gu Incheon:
[7] s4 24 66.6 false :Sasang-gu Sejong:
[8] w1 31 33.3 false :Kangnam-gu Seoul:
[9] w2 32 44.4 true :Dobong-gu Kwangju:
[10] w3 33 55.5 true :Dong-gu Incheon:
[11] w4 34 66.6 false :Sasang-gu Sejong:
[12] a1 41 55.5 true :Dong-gu Incheon:
[13] a2 42 66.6 false :Sasang-gu Sejong:
[14] a3 43 51.9 false :Buk-gu Daejeon:
[15] a4 44 66.6 true :Nam-gu Busan:

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 5
Index to delete? 0
display(): count 15
[0] p1 11 61.1 true :Jong-ro 1-gil, Jongno-gu, Seoul:
[1] p2 12 52.2 false :1001, Jungang-daero, Yeonje-gu, Busan:
[2] p3 13 83.3 true :100 Dunsan-ro Seo-gu Daejeon:
[3] s1 21 65.4 true :Jongno-gu Seoul:
[4] s2 22 54.3 false :Yeonje-gu Busan:
[5] s3 23 55.5 true :Dong-gu Incheon:
[6] s4 24 66.6 false :Sasang-gu Sejong:
[7] w1 31 33.3 false :Kangnam-gu Seoul:
[8] w2 32 44.4 true :Dobong-gu Kwangju:
[9] w3 33 55.5 true :Dong-gu Incheon:
[10] w4 34 66.6 false :Sasang-gu Sejong:
[11] a1 41 55.5 true :Dong-gu Incheon:
[12] a2 42 66.6 false :Sasang-gu Sejong:
[13] a3 43 51.9 false :Buk-gu Daejeon:
[14] a4 44 66.6 true :Nam-gu Busan:

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 5
Index to delete? 14
display(): count 14
[0] p1 11 61.1 true :Jong-ro 1-gil, Jongno-gu, Seoul:
[1] p2 12 52.2 false :1001, Jungang-daero, Yeonje-gu, Busan:
[2] p3 13 83.3 true :100 Dunsan-ro Seo-gu Daejeon:
[3] s1 21 65.4 true :Jongno-gu Seoul:
[4] s2 22 54.3 false :Yeonje-gu Busan:
[5] s3 23 55.5 true :Dong-gu Incheon:
[6] s4 24 66.6 false :Sasang-gu Sejong:
[7] w1 31 33.3 false :Kangnam-gu Seoul:
[8] w2 32 44.4 true :Dobong-gu Kwangju:
[9] w3 33 55.5 true :Dong-gu Incheon:
[10] w4 34 66.6 false :Sasang-gu Sejong:
[11] a1 41 55.5 true :Dong-gu Incheon:
[12] a2 42 66.6 false :Sasang-gu Sejong:
[13] a3 43 51.9 false :Buk-gu Daejeon:

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 5
Index to delete? 14
14: OUT of selection range(0 ~ 13)
Index to delete? 5
display(): count 13
[0] p1 11 61.1 true :Jong-ro 1-gil, Jongno-gu, Seoul:
[1] p2 12 52.2 false :1001, Jungang-daero, Yeonje-gu, Busan:
[2] p3 13 83.3 true :100 Dunsan-ro Seo-gu Daejeon:
[3] s1 21 65.4 true :Jongno-gu Seoul:
[4] s2 22 54.3 false :Yeonje-gu Busan:
[5] s4 24 66.6 false :Sasang-gu Sejong:
[6] w1 31 33.3 false :Kangnam-gu Seoul:
[7] w2 32 44.4 true :Dobong-gu Kwangju:
[8] w3 33 55.5 true :Dong-gu Incheon:
[9] w4 34 66.6 false :Sasang-gu Sejong:
[10] a1 41 55.5 true :Dong-gu Incheon:
[11] a2 42 66.6 false :Sasang-gu Sejong:
[12] a3 43 51.9 false :Buk-gu Daejeon:

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 5
Index to delete? 0
display(): count 12
[0] p2 12 52.2 false :1001, Jungang-daero, Yeonje-gu, Busan:
[1] p3 13 83.3 true :100 Dunsan-ro Seo-gu Daejeon:
[2] s1 21 65.4 true :Jongno-gu Seoul:
[3] s2 22 54.3 false :Yeonje-gu Busan:
[4] s4 24 66.6 false :Sasang-gu Sejong:
[5] w1 31 33.3 false :Kangnam-gu Seoul:
[6] w2 32 44.4 true :Dobong-gu Kwangju:
[7] w3 33 55.5 true :Dong-gu Incheon:
[8] w4 34 66.6 false :Sasang-gu Sejong:
[9] a1 41 55.5 true :Dong-gu Incheon:
[10] a2 42 66.6 false :Sasang-gu Sejong:
[11] a3 43 51.9 false :Buk-gu Daejeon:

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 5
Index to delete? 10
display(): count 11
[0] p2 12 52.2 false :1001, Jungang-daero, Yeonje-gu, Busan:
[1] p3 13 83.3 true :100 Dunsan-ro Seo-gu Daejeon:
[2] s1 21 65.4 true :Jongno-gu Seoul:
[3] s2 22 54.3 false :Yeonje-gu Busan:
[4] s4 24 66.6 false :Sasang-gu Sejong:
[5] w1 31 33.3 false :Kangnam-gu Seoul:
[6] w2 32 44.4 true :Dobong-gu Kwangju:
[7] w3 33 55.5 true :Dong-gu Incheon:
[8] w4 34 66.6 false :Sasang-gu Sejong:
[9] a1 41 55.5 true :Dong-gu Incheon:
[10] a3 43 51.9 false :Buk-gu Daejeon:

============================ PMbyVector Menu =========================
= 0.Exit 1.Display 2.Append 3.Clear 4.Insert 5.Delete 6.Copy 7.Reset =
======================================================================
Menu item number? 0

******************************* Main Menu *********************************
* 0.Exit 1.PersonManager(ch3_2, 4, 6, 7_2, 8, 9, 10)                      *
* 2.Class:Object(ch3_1) 3.CopyConstructor(ch5_1) 4.AllocatedMember(ch5_2) *
* 5.OperatorOverload(ch7_1) 6.Inheritance(ch8) 7.PMbyVector               *
***************************************************************************
Menu item number? 0
Good bye!!
